index.bs

```diff
@@ -1168,6 +1170,20 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator with which to complete the operation.
 </div>
 
+
+#### Privacy considerations #### {#getAssertion-privacy-considerations}
+
+In order to protect the user from being identified without [=user consent|consent=], the client MUST make it impossible for the
+[=[RP]=] to distinguish between the cases:
+
+- A named [=public key credential|credential=] is not available.
+- A named [=public key credential|credential=] is available, but the user denies [=user consent|consent=] to use it.
+
+If the above cases are distinguishable, information is leaked by which a malicious [=[RP]=] could identify the user by probing
+for which [=public key credential|credentials=] are available. In particular, the client MUST NOT return a "{{NotAllowedError}}"
+before the |lifetimeTimer| has expired, as that difference in timing would constitute such an information leak.
+
```

Comment:
IMV, the latter final normative sentence needs to be stated in the algorithm itself, rather than buried down here in Priv Cons.